dotnet/xml/Microsoft.Toolkit.Uwp.UI.Extensions/LogicalTree.xml

```diff
@@ -248,7 +248,7 @@
             Provides a WPF compatible version of TryFindResource to provide a static resource lookup.
             If the key is not found in the current element's resources, the logical tree is then searched element-by-element to look for the resource in each element's resources.
             If none of the elements contain the resource, the Application's resources are then searched.
-            <seealso href="https://docs.microsoft.com/en-us/dotnet/api/system.windows.frameworkelement.tryfindresource" /><seealso href="https://docs.microsoft.com/en-us/dotnet/desktop-wpf/fundamentals/xaml-resources-define#static-resource-lookup-behavior" /></summary>
```

Comment:
Any of the dotnet docs need to be updated in our main repo as they're from the code comments. We should open an issue and fix for 7.1.